bpf/process/types/basic.h

```diff
@@ -1280,6 +1280,78 @@ filter_64ty_map(struct selector_arg_filter *filter, char *args)
 	return 0;
 }
 
+#ifdef __LARGE_BPF_PROG
+FUNC_LOCAL long
+filter_64ty_range(struct selector_arg_filter *filter, char *args)
+{
+	__u64 *v = (__u64 *)&filter->value;
+	int all = 0, i, j = 0;
+
+#pragma unroll
+	for (i = 0; i < MAX_MATCH_VALUES; i++) {
+		__u64 start = v[i * 2], end = v[i * 2 + 1];
+		bool res;
+
+		if (is_signed_type(filter->type))
+			res = ((__s64)start <= *(s64 *)args) && (*(s64 *)args <= (__s64)end);
+		else
+			res = (start <= *(u64 *)args) && (*(u64 *)args <= end);
+
+		if (filter->op == op_in_range && res)
+			return 1;
+		all |= res;
+
+		j += 16;
+		if (j + 8 /* vallen+type */ >= filter->vallen)
```

Comment:
nit: maybe writing `j += sizeof(int64_t) * 2` would make this more explicit and self documented?

using `sizeof(whatever is 64 bits)` instead of 8 maybe is better